### ISSUE TITLE
Improvement: align completion icons on the left side of the screen

### ISSUE THREAD

**Issue by luiggisanso** [dbnschools/moodle-theme_fordson#106]
Hi Chris

I definitivly love your theme
but that's not objective... ;-)

But is it possible to change the alignment of the completion icons just in front of the activities/resources instead of at the end of the line?

I added this code in scss and it works:
.section .activity .actions {
     position: relative; 
}
.path-course-view li.activity span.autocompletion, .path-course-view li.activity form.togglecompletion {
    float: left;
    margin-left: -30px;
    margin-top: -30px;
}

![image](https://user-images.githubusercontent.com/1565841/77332760-c0d93f00-6d22-11ea-9e2c-be46f1e6ed0b.png)

Thanks very for your great work.

**Comment by dbnschools** [comment]
I love this.  Trust me, we would love to have the checkboxes at the beginning of the line.  It is on my list to have it do that.  Possibly by re-writing the renderer.  I am not sure the best approach.  If this css trick works I will consider using it in a future release.  Please know that our schools are closed.  I am working from my basement.  Out top priority is supporting our teachers working from home right now.  I love your willingness to share these awesome improvements!!!

**Comment by dbnschools** [comment]
PS.  Initial testing on my dev site looks really nice!  I'll be honest, after so many years of seeing it on the right side it makes my brain hurt to see it on the left.  lol.  Great job.  This looks like a winner with just a few minutes of testing.

**Comment by luiggisanso** [comment]
Yes I understand, good luck for supporting. We are all in the same case I think...

Cheers.

**Comment by luiggisanso** [comment]
PS WorkPlace have the left icon setting
![image](https://user-images.githubusercontent.com/1565841/77335394-60e49780-6d26-11ea-9096-7e0734dfd035.png)

**Comment by dbnschools** [comment]
I saw some demos of Workplace last summer and I really like their interface and layout. 

I think having it on the left is the way to go.  I might build it into a toggle in Fordson settings to activate or not activate the CSS.  Give people a choice.  

If things slow down for me I will implement this for the next release.

**Comment by luiggisanso** [comment]
_I might build it into a toggle in Fordson settings to activate or not activate the CSS. Give people a choice_

I think it's the perfect choice !

**Comment by dbnschools** [comment]
Thank you.  I think people will love this as an option.

**Comment by luiggisanso** [comment]
FYI Be carfull with Tiles Format

![image](https://user-images.githubusercontent.com/1565841/77337581-5e377180-6d29-11ea-9caa-8d09e12a0a94.png)

And it's only for manual completion

**Comment by luiggisanso** [comment]
Well...

With edit on...
![image](https://user-images.githubusercontent.com/1565841/77338516-c3d82d80-6d2a-11ea-9ec2-2946c6b183bb.png)

Hope you'll be able to check this better than me ;-)

Cheers.

**Comment by luiggisanso** [comment]
Ok just a temporary hack...

![image](https://user-images.githubusercontent.com/1565841/77340197-3e09b180-6d2d-11ea-86d0-f1bcdf00ef74.png)

//* en mode édition on *//
.path-course-view.editing li.activity span.autocompletion img {
    float: left;
    margin-left: -110px;
}

Really not perfect...

Sorry, and good luck ;-)

Bye

**Comment by luiggisanso** [comment]
Hi Chris

Do you have some news about it ?

Cheers.

**Comment by dbnschools** [comment]
I am afraid I couldn't get it stable enough in all scenarios to include.

**Comment by luiggisanso** [comment]
no worries, I know it will work well one day ;-)
